libtock/cycle_counter.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "tock.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DRIVER_NUM_CYCLE_COUNTER 0x00090008
+
+bool cycle_counter_exists(void);
+int cycle_counter_start(void);
+int cycle_counter_stop(void);
+int cycle_counter_reset(void);
+int cycle_counter_read(int *count);
```

Comment:
It's probably best to document these in the header file.

It's also worth saying that stop does not reset the counter and it needs to manually be reset